cmake/Findcryptopp.cmake

```diff
@@ -1,21 +0,0 @@
-find_path(CRYPTOPP_INCLUDE_DIR keccak.h
```

Comment:
@no2chem  strange why this is showing up in my diff, since I see you have deleted it twice already.

Reply:
It's probably a macos casing issue. I think the file was renamed FindCryptopp.cmake

Reply:
Oh nevermind, just some artifact from rebasing off different branches, cleared it up.